src/app/index.route.js

```diff
@@ -15,4 +16,11 @@
     });
   }
 
+  /** @ngInject */
+  function runBlock($rootScope) {
+    $rootScope.$on('$stateChangeSuccess', function(e, toState) {
+      $rootScope.ACTIVE_STATE = toState;
```

Comment:
1. See https://github.com/jgeewax/phoenix/pull/45/files#r45142055
2. Per the Angular community styleguide, we should only be putting one operation (run/config/etc.) per file, this would probably be better placed within https://github.com/jgeewax/phoenix/blob/master/src/app/index.run.js